README.md

```diff
@@ -14,4 +14,132 @@ Some of the features include:
 
 For more in-depth information on the CanoE-COACH visit [our website](https://ay2021s1-cs2103-f10-1.github.io/tp/).
 
+1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+   Some example commands you can try:
+
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
```

Comment:
Might want to fetch the updated UserGuide.md to paste here, as it was edited and merged yesterday to resolve some docu bugs (e.g. address here is removed)

Reply:
yup done